realm/realm-library/build.gradle

```diff
@@ -46,6 +46,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    compile 'com.android.support:recyclerview-v7:23.1.1'
```

Comment:
We still need to do something about this as pulling in this dependency automatically is a no. I'll try to play around with it .